mailpoet/lib/Automation/Engine/API/WorkflowResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types = 1);
```

Comment:
This whole thing is probably unnecessary. I thought it would be nice to avoid the duplication we have throughout the app though.

Reply:
We absolutely need something like this! I was thinking about creating `Mappers` but this looks also nice. Not sure which is better. We may need to reuse some smaller units (such as building only workflow steps) so maybe some more granular mappers will be better than full responses.

Reply:
I think some kind of `Mappers` would probably be nice. Maybe we could even set them up in such a way that API requests can include which data they want in the response, potentially speeding things up. For now I'm thinking I might revert this change in large part because it doesn't seem worth it to change premium just for this.